README.md

```diff
@@ -41,6 +41,25 @@ For an even easier usage, just `#import <KVOController/NSObject+FBKVOController.
 [self.KVOController observe:clock keyPath:@"date" options:NSKeyValueObservingOptionInitial|NSKeyValueObservingOptionNew action:@selector(updateClockWithDateChange:)];
 ```
 
+## Swift
+
+KVOController works fine with Swift but the properties that you need to observer must be marked as `dynamic` check the following example:
+```Swift
+class TasksListViewModel: NSObject, TasksListViewModelProtocol{
+
+    dynamic var tasksList: [TaskList] = []
+}
+
+/// In ViewController.swift
+
+import KVOController
+
+self.KVOController.observe(self.viewModel as? AnyObject, keyPath: "tasksList", options: [.New, .Initial]) { (viewController, viewModel, change) in
```

Comment:
Any chance we can line break these as follows to improve readability on small screens?
```swift
kvoController.observe(self,
                                      keyPath: "a",
                                      options: [.new, .initial]) { (viewController, viewModel, change) in
  self.taskListsTableView.reloadData()
}
```

Also, there is no need for `self.KVOController`. Only `kvoController` (lowercase, per Swift 3).
Also, it's `.new, .initial` per Swift 3.

And, frankly, I am not sure whether there is a need for `as? AnyObject`, since it should always pass...